inference-engine/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp

```diff
@@ -143,9 +144,16 @@ std::shared_ptr<ngraph::Node> makeCTCGreedyDecoder(
 
 std::shared_ptr<ngraph::Node> makeCTCGreedyDecoderSeqLen(
         const ngraph::Output<Node>& inputData,
+        const ngraph::Output<Node>& sequenceLength,
         int blankIndex,
         bool mergeRepeated,
-        const element::Type& idxPrec);
+        const element::Type& idxPrecision = element::i32);
+
+std::shared_ptr<ngraph::Node> makeCTCGreedyDecoderSeqLen(
```

Comment:
Should it be keeped?

Reply:
Those are helper functions. I guess they could be used somewhere else. Maybe.

Reply:
Removed